developer_manual/app/code_signing.rst

```diff
@@ -44,6 +44,11 @@ Is Code Signing Mandatory For Apps?
 Code signing is optional for all third-party applications. 
 Applications with a tag of "Official" on https://marketplace.owncloud.com/ require code signing.
 
+Can I omit Code Signing for personal use and/or testing?
```

Comment:
Can you change the case of the title to be: _"Can I Omit Code Signing for Personal Use and/or Testing?"_ pls?